workshops/socialauth.py

```diff
@@ -0,0 +1,76 @@
+from django.http import HttpResponseForbidden
+
```

Comment:
Please conform to our styling:

1. sorted alphabetically
2. grouped by dependency, ie. system modules in the first group, related dependencies (e.g. Django, or selectable) in the second group, and project imports in the third group.